archlinuxcn/devtools-archlinuxcn/PKGBUILD

```diff
@@ -49,6 +60,10 @@ build() {
 package() {
   cd ${_pkgname}-${pkgver}
   make PREFIX=/usr DESTDIR="${pkgdir}" install
+
+  for _arch in aarch64 armv7h armv6h arm; do
```

Comment:
At first I wonder what `arm` really means, and later I understand it means armv5 on archlinuxarm.org. Better to have a comment here.